src/app/store.ts

```diff
@@ -1,6 +1,10 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
```

Comment:
The function `combineSlices` is incorrect. You should use `combineReducers` from `@reduxjs/toolkit` to combine your slices into a root reducer.